docker/setup/postinstall.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+display_error() {
+	echo " !--> ERROR on postinstall:"
+	tail -n 200 /tmp/dockerize.log
+	exit 1
+}
+
+echo " ---> POSTINSTALL"
+
+# Add MySQL-to-Postgres migration script to path (used in entrypoint.sh)
+cp ./docker/mysql-to-postgres/bin/migrate-mysql-to-postgres /usr/local/bin/
+
+# Ensure we can write in /tmp/op_uploaded_files (cf. #29112)
+mkdir -p /tmp/op_uploaded_files/ && chown -R $APP_USER:$APP_USER /tmp/op_uploaded_files/
+
+cp ./packaging/conf/database.yml ./config/database.yml
+
+if test -f ./docker/setup/postinstall-$PLATFORM.sh ; then
+	echo " ---> Executing postinstall for $PLATFORM..."
+	./docker/setup/postinstall-$PLATFORM.sh
+fi
+
+echo " ---> Precompiling assets. This will take a while..."
+
+(
+	pushd "${APP_PATH}/frontend"
+
+	export NG_CLI_ANALYTICS=ci # so angular cli doesn't block waiting for user input
+
+	# Installing frontend dependencies
+	RAILS_ENV=production npm install
+
+	popd
+
+	# Bundle assets
+	su - postgres -c "$PGBIN/initdb -D /tmp/nulldb"
```

Comment:
FYI this change means we can get rid of the dynamically provisioned DB we use in some of our docker compilation jobs.